src/subdomains/supporting/support-issue/support-issue.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { IEntity } from 'src/shared/models/entity';
+import { Column, Entity, ManyToOne } from 'typeorm';
+import { Transaction } from '../payment/entities/transaction.entity';
+
+export enum SupportIssueState {
+  CREATED = 'Created',
+  PENDING = 'Pending',
+  COMPLETED = 'Completed',
+}
+
+export enum SupportIssueReason {
+  FUNDS_NOT_RECEIVED = 'FundsNotReceived',
+  OTHER = 'Other',
+}
+
+@Entity()
+export class SupportIssue extends IEntity {
+  @Column({ length: 256, default: SupportIssueState.CREATED })
+  state: SupportIssueState;
+
+  @Column({ length: 256, nullable: false })
+  reason: SupportIssueReason;
+
+  @Column({ length: 'MAX' })
+  description: string;
+
+  @Column({ length: 256, nullable: true })
+  fileUrl: string;
+
+  @ManyToOne(() => Transaction, (transaction) => transaction.supportIssues, { nullable: false, eager: true })
+  transaction: Transaction;
+}
```

Comment:
We will have other issues in the future
- Add a type column (with `TransactionIssue` for now)
- `transaction` relation is one to one
- `transaction` is nullable